sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -178,18 +160,13 @@ export class EventPosition {
     }
 
 // @public
-export interface EventProcessorCommonOptions {
+export interface EventProcessorOptions {
     defaultEventPosition?: EventPosition;
-    maxBatchSize?: number;
-    maxWaitTimeInSeconds?: number;
+    maxBatchSize: number;
+    maxWaitTimeInSeconds: number;
```

Comment:
Just curious, If `maxBatchSize` and `maxWaitTimeInSeconds` are in `EventProcessorOptions` interface then why these two are not optional?

Reply:
The event processor basically became an internal class with the introduction of EventHubConsumerClient.

So my concern was just making sure we weren't doing some sort of default values in lower parts of the stack - those kinds of decisions should be done as close to the surface as possible.